docs/tarball.md

```diff
@@ -12,14 +12,23 @@ oci_tarball(
 )
 ```
 
-and then run it in a container like so:
+and then load and run it in a container like so:
 
 ```
-bazel build //path/to:tarball
-docker load --input $(bazel cquery --output=files //path/to:tarball)
-docker run --rm my-repository:latest
+bazel run :tarball
+docker run --rm "my-repository:latest"
 ```
 
+By default, the generated script will attempt to use Docker or Podman to load the image. If you need to use a runtime other than Docker or Podman, you can replace the entire script template:
+
+```
+oci_tarball(
+    name = "tarball",
+    image = ":image",
+    repo_tags = ["my-repository:latest"],
+    run_template = "//path/to/your.tpl",
```

Comment:
I'd be happier if we didn't introduce yet another attribute here. if the default behavior doesn't serve the specific use cases, they can always wrap oci_tarball in a macro and their own runnable target.

Reply:
I made the attribute private for now, and filed #295 where we can consider what public API surface might be appropriate to provide more control.